policyengine_us/reforms/states/ri/ctc/ri_ctc_reform.py

```diff
@@ -0,0 +1,192 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+from policyengine_core.periods import instant
+
+
+def create_ri_ctc() -> Reform:
+    class ri_ctc_eligible_children(Variable):
+        value_type = int
+        entity = TaxUnit
+        label = "Rhode Island CTC eligible children count"
+        unit = "#"
+        definition_period = YEAR
+        defined_for = StateCode.RI
+
+        def formula(tax_unit, period, parameters):
+            p = parameters(period).gov.contrib.states.ri.ctc
+            if not p.in_effect:
+                return 0
+
+            person = tax_unit.members
+            age = person("age", period)
+            is_dependent = person("is_tax_unit_dependent", period)
+
+            # Check age eligibility
+            meets_age = age < p.age_limit
+
+            eligible = is_dependent & meets_age
+            return tax_unit.sum(eligible)
+
+    class ri_ctc_maximum(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Rhode Island CTC maximum amount before phaseout"
+        unit = USD
+        definition_period = YEAR
+        defined_for = StateCode.RI
+
+        def formula(tax_unit, period, parameters):
+            p = parameters(period).gov.contrib.states.ri.ctc
+            if not p.in_effect:
+                return 0
+
```

Comment:
remove the if in effect = 0 everywhere, we will adjust the list of rhode island credits which will only be applied if the in effect is true.

Reply:
applies to all variables